uat/features/admin_menu.feature

```diff
@@ -23,7 +24,7 @@ Feature: Administration Menu
     And the "Account links" checkbox should be checked
     And the "Shortcuts" checkbox should not be checked
 
-  @api @javascript
+  @api @javascript @deploy @safe
```

Comment:
I agree it is not recommended to do so on production but it is a safe operation even if it does degrade performance.